src/kakarot/accounts/uninitialized_account.cairo

```diff
@@ -6,50 +6,38 @@ from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin
 from starkware.cairo.common.alloc import alloc
 from starkware.starknet.common.syscalls import get_caller_address, replace_class, library_call
 
-// We are intentionally causing a storage_slot collision here,
-// by defining these variables in both `uninitialized_account` and `account_contract`.
-// We are defining them here instead of in the account library, so as to not depend
-// on content of the account library in uninitialized_account and ensure a fixed class hash.
-@storage_var
-func Account_evm_address() -> (evm_address: felt) {
-}
-
-@storage_var
-func Account_kakarot_address() -> (kakarot_address: felt) {
+@contract_interface
+namespace IKakarot {
+    func get_account_contract_class_hash() -> (account_contract_class_hash: felt) {
+    }
 }
 
 const INITIALIZE_SELECTOR = 0x79dc0da7c54b95f10aa182ad0a46400db63156920adb65eca2654c0945a463;  // sn_keccak('initialize')
 
 // @title Uninitialized Contract Account. Used to get a deterministic address for an account, no
 // matter the actual implementation class used.
+
+// @notice Deploys and initializes the account with the Kakarot and EVM addresses it was deployed with.
```

Comment:
Apply the same rules as for python files. Rules are defined by the ruff linter, see

https://docs.astral.sh/ruff/rules/#pydocstyle-d

and more precisely here

https://docs.astral.sh/ruff/rules/non-imperative-mood/

> First line of docstring should be in imperative mood: "{first_line}"

Reply:
but kakarot is de facto a python project and we enforce this linter for python files
it's opinionated but we need to be consistent with half of the codebase